tools/preload.py

```diff
@@ -63,6 +63,8 @@ def get_absolute_url(domain, path, icon):
 
     if(path.startswith('http')):
         return icon_path
+    elif path_absolute_from_app_root:
```

Comment:
Yep, thanks for the clarification. I think I understood `path_absolute_from_app_origin`, which is exactly the contrary :-)